bin/Powheg/Templates/createTarBall_template.sh

```diff
@@ -39,10 +39,22 @@ sed -i "s/^numevts.*/numevts NEVENTS/g" powheg.input
 sed -i "s/^iseed.*/iseed SEED/g" powheg.input
 grep -q "withnegweights" powheg.input; test $$? -eq 0 || printf "\nwithnegweights 1\n" >> powheg.input
 
-# turn into single run mode
-sed -i "s/^manyseeds.*/#manyseeds 1/g" powheg.input
-sed -i "s/^parallelstage.*/#parallelstage 4/g" powheg.input
-sed -i "s/^xgriditeration/#xgriditeration 1/g" powheg.input
+ISVRES="false"
+grep -qi "powhegboxRES" $$WORKDIR/$$folderName/VERSION ; test $$? -ne 0  || ISVRES="true"
+# For Powheg vRES, if the gridpack has been produced in several stages (using manyseeds 1),
+# we need to turn on manyseeds & parallelstage also for the final gridpack.
+# we test whether manyseeds has been used by checking if "pwgubound-0001.dat" exists
+if [ $$ISVRES == "true" -a -f "$$WORKDIR/$$folderName/pwgubound-0001.dat" ]; then
```

Comment:
we shouldn't require 0001 here but just make sure 00x is there. maybe we can use the logic from this pr: 
https://github.com/cms-sw/genproductions/pull/3415/files

Reply:
I changed it to check whether more than one file matching `pwgubound-*.dat` exists. As far as I can tell, I cannot simply copy the logic from the PR you linked since that would also catch an `pwgubound.dat` file from a single-process production (without `manyseeds`).